<?xml version="1.0" encoding="UTF-8"?>

<!--
SPDX-FileCopyrightText: 2019-2023 Mattia Basaglia <dev@dragon.best>

SPDX-License-Identifier: GPL-3.0-or-later
-->

<ui version="4.0">
 <class>glaxnimate::gui::font::FontSizeWidget</class>
 <widget class="QWidget" name="glaxnimate::gui::font::FontSizeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDoubleSpinBox" name="spin_size">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="view_size"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>on_size_edited(double)</slot>
 </slots>
</ui>
